### DISCUSSION TITLE
Ideas for new goals

### DISCUSSION THREAD

**Discussion by Gaming32** [Gaming32/bingo#1] [Ideas]
Post ideas for goals here, and vote on the ones you like!

**Comment by samnrad** [comment]
very hard

>~~three different armor trims~~
>tame every possible animal
>collect one of every crop
>~~trade with a wandering villager~~
>~~win a raid~~
>~~collect all the goat horns~~
>obtain every item with wood in its crafting recipe
>obtain every item with iron in its crafting recipe
>collect all three froglights
>breed sniffers
>name a left handed stray
>kill a warden with zombified piglins


hard
>obtain an ender chest
>die to a spider jockey
>have three potion effects at once
>~~cure a villager~~
>bring a piglin brute into the overworld
>get one of every boat
>get a mule
>kill an entity with thorns
>mine a reinforced deepslate
>blow up an end crystal
>wear frost walker boots
>reach y level 350
>obtain an oxidized copper block
>obtain mangrove wood
>break a silverfish infested block
>visit 35 overworld biomes
>kill a piglin with only a zombified piglin
>collect every horse armor
>kill an illager of any type

medium
>ride a pig
>kill 10 bats
>get a sticky piston
>kill a creeper with tnt
>find a heart of the sea
>fill a full map in the overworld
>~~bonemeal a mushroom~~
>travel 5km in a boat
>~~build a snowman~~
>~~build an iron golem~~
>freeze to death
>kill five baby mobs
>kill five zombie villagers
>kill five witches

easy
>survive an otherwise deadly fall with a haybale
>ignite a creeper using flint and steel
>~~break a mob spawner~~
>obtain cherry log
>obtain a cobweb
>obtain dolphin's grace
>turn deepslate to moss
>obtain diorite, granite, and andesite
>obtain one netherrack

very easy
>die to fall damage
>die to cactus
>kill ten mobs

anti spots:

very hard:
>never have anyone on your team fall below five hearts
>never press the s key/walk backwards
>no tools other than pickaxes and swords
>no enchantments
>no jumping/spacebar
>no trading with villagers or bartering with piglins
>no item stacks above 8 in your inventory


hard:
>never have anyone on your team die
>no boats/minecarts/riding any mob
>never take drowning/suffocating damage
>never take burning damage


medium:
>never fall below 3 bars of hunger

very easy:
>never eat

**Reply by Earthcomputer** [reply]
> three different armor trims

Might have to increase this number, as you can get armor trims from a lot of different places. Not sure, needs playtesting.

> collect one of every crop

Which crop makes this very hard?

> trade with a wandering villager

There is already a "kill a wandering trader" goal under "hard", so it should be the same difficulty as that. Maybe that should be moved to "very hard" though.

> win a raid

This goal already exists under "hard", with the name "become a hero of the village".

> kill a warden with zombified piglins

That's extremely hard. How do you envisage players getting enough zombified piglins for that? I think maybe just a "kill a warden" goal under an easier category.

> cure a villager

Goal already exists under "hard".

> reach y level 350

I'm guessing you're thinking of TNT, not elytra? This is too high to be launched by slime blocks.

> obtain an oxidized copper block

This borders on "very hard", oxidization takes ages. You could place a lot of copper blocks at the start though and wait for one of them to oxidize throughout, but this may not happen for the duration of a "hard" bingo, hence why I'm thinking it may need to be "very hard". Needs playtesting.

> visit 35 overworld biomes

How many rare biomes would this require you to find? Would this goal's progress be shared across all team members, unlike other goals? How would players be able to track the progress of this goal? Unlike the adventuring time advancement where it's not the main objective of the game, I think players need a way to track this one.

> fill a full map in the overworld

This takes quite a long time without an elytra. Might need to be moved to "hard" for this reason. Needs testing.

> bonemeal a mushroom

Goal already exists under "easy".

> build a snowman

Goal already exists under "easy".

> build an iron golem

Goal already exists under "medium".

> break a mob spawner

Goal already exists under "hard". Should probably be moved to medium or easy I think, spawners aren't that rare if you know where to look for them.

> never have anyone on your team fall below five hearts

This is almost too hard for this category, ironically being easiest for the "very easy" category. I don't imagine this goal would ever be achieved in practice, at least in any of our playtests.

> never press the s key/walk backwards

Not sure how easy this is to detect on the server, maybe it's possible though.

> no enchantments

This isn't very hard at all. Given that it would conflict with any goal that requires an enchanted item it would always be in a row/column/diagonal where it is easy to achieve. Better would be "no enchanted books", forcing players to enchant via the enchanting table or by finding enchanted items by exploring.

> no jumping/spacebar

Too annoying.

> no item stacks above 8 in your inventory

Way too hard to avoid imo.

> never have anyone on your team die

Similar point to the one above about 5 hearts, and although it *is* more achievable, I don't see any full team managing to achieve this in any of our playtests.

**Reply by Gaming32** [reply]
> fill a full map in the overworld

Additionally, this goal already exists under "hard"

**Reply by samnrad** [reply]
> three different armor trims

i dunno, put it in and see how bad it is

> collect one of every crop

sniffer crops and chorus plant

> kill a warden with only zombified piglins

named warden 50 blocks out + pig farm + lightning rod or channeling (a pain but in a very hard this is not too much setup

> reach y level 350

i can think of like 5 different ways to do this, all with varying levels of difficulty

> obtain oxidized copper block

agreed, obviously it's how many you can throw down and wait for one to turn.

> visit 35 overworld biomes

the achievement for visiting all overworld biomes is 53 biomes, so i think 35 is a decent amount. needs playtesting

> never have anyone on your team fall below five hearts

i mean we have dont craft sticks as an anti spot, i think thats basically impossible to avoid

> never have anyone on your team below five hearts

maybe move to very easy, since the games are so fast it would really be a gimme unless you screwed up

> never have anyone on your team die

same idea as above

> no item stacks above 8 in your inventory

increase to 32 or put in very easy category

> no jumping/spacebar

idk put in very easy i guess

**Reply by Earthcomputer** [reply]
> i mean we have dont craft sticks as an anti spot, i think thats basically impossible to avoid

I don't think having precedent in the goals we inherited is a good reason to make more of these goals. On the contrary, we probably need to look into changing or removing that goal if people see it as unachievable.

**Comment by Earthcomputer** [comment]
Rebalance netherite item based goals in lieu of the fact that netherite templates are needed now.

**Comment by ghost** [comment]
- (Medium?) Fall out of the world
- (Medium) Break an unbreakable block
- (Any) Mine X of Y
- (Any) Kill X mobs (of a specific type?)
- (Easy) Break a boat into sticks (possibly a bad goal as it's glitch specific and pretty much just needs knowing how to do it)
- (Any) Generic "never get X advancement" / "get X advancement" goals
- (Hard) Break a diamond tool, weapon or armor piece
- (Any) Eat at least X different items
- (Any) Some form of villager-trading-related goal (buy x items / get x emeralds / level x villagers up to y level)
- (Hard) Get a music disk from a creeper
- (Very Hard) Summon a wither (indirectly covered by wither roses and I believe beacon)
- (Any) Have at least X potion effects at once
- (Very Hard, requires a mooshroom island and lightning) Drink every kind of suspicious stew
- (Medium) Collect every kind of flower (similar to the above)
- (Medium) Wear 4 different colors of leather armor
- (Hard) Get a dog with every color of collar
- (Medium) Make a jeb sheep
- (Hard / Very Hard) Cure a naturally spawned zombie villager
- (Medium / Hard) Obtain X item from a piglin
- (Any - Might already exist) Convert a piglin to a zombified piglin / hoglin to a zoglin
- (Easy / Medium) Drown a zombie
- (Medium / Hard) Drown a husk
- (Easy / Medium) Get an iron ingot / carrot / potato from a zombie or a copper ingot from a drowned
- (Any) Sail a boat X metres
- (Medium / Hard) Break a carrot on a stick / warped fungus on a stick
- (Medium) Equip a curse of binding item
- (Easy) Look at an enderman while wearing a carved pumpkin
- (Hard) Wear any mob head
- (Medium / Hard - Already exists I believe) Point a compass to a lodestone

**Reply by Earthcomputer** [reply]
> (Medium) Break an unbreakable block

Good but the easiest way of doing it involves lagging the server, which ruins the experience for the other team.

> (Any) Mine X of Y
> (Any) Kill X mobs (of a specific type?)
> (Any) Generic "never get X advancement" / "get X advancement" goals

Too general.

> (Hard) Break a diamond tool, weapon or armor piece

Medium, doesn't need many diamonds and a few ways to do quickly.

> (Any) Some form of villager-trading-related goal (buy x items / get x emeralds / level x villagers up to y level)

Too general, although I do like the idea of emerald blocks being a goal.

> (Medium) Collect every kind of flower (similar to the above)

I'm assuming given the difficulty not including wither roses.

> (Hard / Very Hard) Cure a naturally spawned zombie villager

Goal already exists under "hard", except for the "naturally spawned" requirement which imo doesn't add much.

> (Medium / Hard) Obtain X item from a piglin

Good idea but too general.

> (Medium / Hard) Break a carrot on a stick / warped fungus on a stick

Carrot on a stick one already exists under "hard", could add a warped fungus on a stick version.

> (Hard) Wear any mob head

Does this include wither skulls? If not we already have an obtain mob head goal under "very hard" so it's basically the same as that, although I think this is more fun.

**Reply by ghost** [reply]
The point of the goals was to be very general, with specific things in the blanks as an extension to some of the existing generic goals (e.g. there's goals for like "get 15 melon slices" or whatever).

> Does this include wither skulls? If not we already have an obtain mob head goal under "very hard" so it's basically the same as that, although I think this is more fun.

Yes, I would suggest it as more of a replacement because I consider that goal to be pretty much a never-go-for-it right now, unless you feel like exploring for an ancient city for ages. Alternatively we could add a mod that makes thunder and rain significantly more common.

> Goal already exists under "hard", except for the "naturally spawned" requirement which imo doesn't add much.

Ah alright

> Carrot on a stick one already exists under "hard", could add a warped fungus on a stick version.

I would suggest merging them, probably.

> Good but the easiest way of doing it involves lagging the server, which ruins the experience for the other team.

Requires client mods to work without a fuck ton of redstone, and is therefore not really feasible anyway. Intention was to do it with TNT.

**Reply by Gaming32** [reply]
For the unbreakable blocks, I don't see how that causes lag, requires a client mod, or requires a lot of redstone. Isn't it just the simple structure with two TNT, a piston, a trapdoor, and a few blocks?

**Reply by ghost** [reply]
The way to do it that requires the least resources is laghammer, but I've realized he's wrong anyway. For laghammer to work you have to instamine the puston, which is only possible with a diamond/netherite eff 5 pickaxe and haste 2

**Comment by samnrad** [comment]
very hard
>obtain every item with the letter z in its name
>collect all items with rat in their name
>obtain all blocks with block in their name

hard
>no sneaking
>obtain every item with redstone in its crafting recipe
>obtain every item with the letter j in its name
>obtain every grass top (grass block, podzol, mycelium, warped and crimson nylium)
>collect all blocks with moss in their name
>obtain all blocks with car in their name
>collect all items with bra in their name
>obtain all items with "aw" in their name

medium
>a stack of azalea leaves
>die to poison
>obtain every item with the letter q in its name
>collect all the minecarts
>get an armor drop from a mob
>craft a stack of brick blocks

**Reply by Earthcomputer** [reply]
Some of these have more items that can fit in the inventory

**Reply by ghost** [reply]
> no sneaking

This goal just seems kinda like, bullshit. I don't like the "no xxx" goals because it's too easy to accidentally do a lot of them and lose your entire strategy.

> die to poison

Uh how

> Get an armor drop from a mob

This is not medium idt

Also the various "collect all X with Y in their name" seem a bit weird, especially because of languages/translations and because it's not like "get every item with bra in the name" is a very understandable goal.

**Comment by samnrad** [comment]
>obtain all the buckets
>never go below y0
>obtain a level 5 enchantment
>no buckets
>make x tinted glass
>obtain all three shards (prismarine, echo, and amethyst)
>survive a melee hit from a warden
>kill a skeleton using only other skeletons
>die to llama spit
>kill a mob other than a ghast with a fireball
>kill every flying mob except the ender dragon (allay, bat, bee, blaze, chicken, ghast, parrot, phantom, vex)
>kill 5 glow squid
>collect all 4 eggs (chicken egg, sniffer egg, dragon egg, and turtle egg)
>kill a zoglin
>make a signal campfire
>use milk to remove a potion effect
>sift through 10 suspicious gravel/sand
>kill 30 villagers
>obtain a blue axolotl in a bucket
>walk 1000m on a strider
>trade with 20 different villagers
>use a grindstone
>craft all the tables
>no food items that can be crafted
>kill a polar bear

**Reply by JmonJoshMC** [reply]
Wouldn't the **Never Go Below Y0** goal make other ones impossible? Like any that involve wardens/ancient cities? 
Besides that, these ideas probably work well.

**Reply by Earthcomputer** [reply]
You could make the never go below Y0 goal conflict with those goals but yeah

**Reply by cputnam-a11y** [reply]
Could you not kill the warden with tnt from above y0?
